api/test/zero/unit/aou_module_test.clj

```diff
@@ -23,43 +24,49 @@
-                                            :Arrays.cluster_file
-                                            :Arrays.call_rate_threshold
-                                            :Arrays.haplotype_database_file]]
+        all-expected-keys-no-control       (sort [:Arrays.preemptible_tries
+                                                  :Arrays.environment
+                                                  :Arrays.ref_dict
+                                                  :Arrays.params_file
+                                                  :Arrays.subsampled_metrics_interval_list
+                                                  :Arrays.chip_well_barcode
+                                                  :Arrays.sample_alias
+                                                  :Arrays.variant_rsids_file
+                                                  :Arrays.ref_fasta_index
+                                                  :Arrays.dbSNP_vcf
+                                                  :Arrays.disk_size
+                                                  :Arrays.contamination_controls_vcf
+                                                  :Arrays.green_idat_cloud_path
+                                                  :Arrays.fingerprint_genotypes_vcf_index_file
+                                                  :Arrays.vault_token_path
+                                                  :Arrays.reported_gender
+                                                  :Arrays.dbSNP_vcf_index
+                                                  :Arrays.extended_chip_manifest_file
+                                                  :Arrays.zcall_thresholds_file
+                                                  :Arrays.sample_lsid
+                                                  :Arrays.red_idat_cloud_path
+                                                  :Arrays.gender_cluster_file
+                                                  :Arrays.ref_fasta
+                                                  :Arrays.bead_pool_manifest_file
+                                                  :Arrays.analysis_version_number
+                                                  :Arrays.fingerprint_genotypes_vcf_file
+                                                  :Arrays.cluster_file
+                                                  :Arrays.call_rate_threshold
+                                                  :Arrays.haplotype_database_file])
+        all-expected-keys                   (sort (concat all-expected-keys-no-control
+                                                          [:Arrays.control_sample_vcf_index_file
+                                                           :Arrays.control_sample_intervals_file
+                                                           :Arrays.control_sample_vcf_file
+                                                           :Arrays.control_sample_name]))]
```

Comment:
Seems like we could compute this, but maybe we don't wanna?

Reply:
yeah, we could derive this, i was just being lazy and in a hurry so didn't do so.

Reply:
OK by me.